src/dragonball/Cargo.toml

```diff
@@ -39,6 +39,10 @@ thiserror = "1"
 vmm-sys-util = "0.10.0"
 virtio-queue = { version = "0.4.0", optional = true }
 vm-memory = { version = "0.9.0", features = ["backend-mmap"] }
+nydus-storage = { git = "https://github.com/dragonflyoss/image-service.git", rev = "e429be3e8623d47db0f97186f761aeda2983c6f4", features = ["backend-registry","backend-oss"]}
+# As feature backend-registry and backend-oss is not enabled in dbs crate, just enable it here temporarily  
+openssl = { version = "0.10.35", features = ["vendored"] }
+# With `openssl-src`, openssl wil be compiled statically and we don't have to install openssl-dev manually anymore.
```

Comment:
Why need this?
Move it to runtime-rs/hypervisor?

Reply:
Thanks. I will move it there